docs/concepts/glossary.md

```diff
@@ -3,6 +3,9 @@
 ## Abstract Syntax Tree
 A tree representation of the syntactic structure of source code. Each tree node represents a construct that occurs. The tree is abstract because it does not represent every detail appearing in the actual syntax; it also does not have a standard representation.
 
+## Backfill
```

Comment:
Don't we currently use this term for initial loading as well? As in "we are backfilling the model we just added."

Reply:
Updated. Let me know what you think